Topology Optimization/Filters/Filter.m

```diff
@@ -23,25 +23,25 @@
                 case 'P1'
                     switch settings.optimizer
                         case {'MMA','PROJECTED GRADIENT','IPOPT'}
-                            obj = Filter_P1_Density;
+                            obj = Filter_P1_Density();
```

Comment:
Only curious, does this have any benefit?

Reply:
No change in code's behaviour, but it is a good practice to put the brackets when calling a function (the class's constructor in this case). I just passed by to check out something and applied a little Boys' Scouts rule.